tools/mo/openvino/tools/mo/ops/unique.py

```diff
@@ -51,6 +51,21 @@ def supported_attrs(self):
 
     @staticmethod
     def infer(node: Node):
+        version = node.soft_get("version", "experimental")
+        if version == "opset10":
+            Unique.infer_10(node)
+        else:
+            Unique.infer_experimental(node)
+
+    @staticmethod
+    def infer_10(node: Node):
+        input_shape = node.in_port(0).data.get_shape()
+        for out_port in node.out_ports():
+            if node.is_out_port_connected(out_port):
+                node.out_port(out_port).data.set_shape(input_shape)
```

Comment:
shapes of other outputs are not necessarily the same as the input shape. please check